src/pages/ListScholarships.jsx

```diff
@@ -56,38 +58,41 @@ function ListScholarships() {
       </Helmet>
       <Drawer
         sx={{
-          display: { xs: 'none', md: 'block' },
           flexShrink: 0,
           width: drawerWidth,
           '& .MuiDrawer-paper': {
             width: drawerWidth,
             position: 'absolute',
             boxSizing: 'border-box',
           },
-          position: 'sticky',
+          position: { xs: 'static', md: 'sticky' },
           overflowY: 'auto',
         }}
-        variant="permanent"
+        open={drawerOpen || isDesktop}
+        variant={isDesktop ? 'permanent' : 'temporary'}
         anchor="left">
-        <FilterPanel />
+        <FilterPanel onClose={() => setDrawerOpen(false)} />
       </Drawer>
-      <Container
-        maxWidth="md"
-        component="main"
-        ref={ref}
+
+      <Box
         sx={{
-          bgcolor: 'background.default',
-          flexGrow: 1,
-          padding: 2,
-          position: 'sticky',
-          overflowY: { md: 'auto' },
+          width: '100%',
+          display: 'flex',
+          flexDirection: 'column',
```

Comment:
allows `FilterBar` to scroll with the `scholarshipList`

Reply:
That's because the `display` was set to `flex`. You don't need `display` or `flexDirection` here. Just `overflowY: auto` suffices (as long as `overflowY` is removed from the Container below).